src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/cloudconfigs/fdm/salesforce-cloud-service/.content.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:lastModified="{Date}2020-12-21T22:03:43.978+05:30"
+        cq:lastModifiedBy="admin"
+        jcr:lastModified="{Date}2020-12-21T22:03:43.915+05:30"
+        jcr:lastModifiedBy="admin"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Salesforce Cloud Config"
+        sling:resourceType="fd/fdm/gui/components/admin/fdmcloudservice/rest"
+        access_token_uri="https://login.salesforce.com/services/oauth2/token"
+        authenticationType="OAuth 2.0"
+        dataSourceType="REST"
+        filename="salesforce_swagger.json"
+        location="header"
+        name="salesforce-cloud-service"
```

Comment:
Even for using term salesforce there was some discussion. Please check with PM and confirm name to be use

Reply:
name is the node name. All the user facing titles of FDM and Data Sources has been approved by PM.